### ISSUE TITLE
Why is my loss in the verification set going up but my loss in the training set going down

### ISSUE THREAD

**Issue by 826zhanji** [adamklec/copynet#7]
excuse me，after i run your code in my dataset，i found the va_loss is going up ，but train loss is going down.so
So，Have you ever encountered this situation when working with your own data set.
Thanks for sharing

**Comment by Akhila-Yerukola** [comment]
It seems that you're overfitting on your train set. Either add regularization or do early stopping. There are lots of other ways to prevent overfitting as well.

**Comment by zhhhzhang** [comment]
> excuse me，after i run your code in my dataset，i found the va_loss is going up ，but train loss is going down.so
> So，Have you ever encountered this situation when working with your own data set.
> Thanks for sharing

hi! i wonder if the task of implement is for text summarization?  could you share a toy_dataset so that i can make my own dataset? just an data example? thanks
